src/templates/components/healthServices/healthServices.stories.ts

```diff
@@ -0,0 +1,54 @@
+import { Meta, StoryObj } from '@storybook/react'
+import HealthServices from './'
+import { HealthServices as FormattedHealthServices } from '@/types/formatted/healthServices'
+
+const meta: Meta<typeof HealthServices> = {
+  title: 'Components/Health Services',
+  component: HealthServices,
+}
+export default meta
+
+type Story = StoryObj<typeof HealthServices>
+
+const sampleServices: FormattedHealthServices = [
+  {
+    name: 'PTSD care',
+    vetCenterTypeOfCare: 'counseling',
+    vetCenterFriendlyName: null,
+    alsoKnownAs: null,
+    vetCenterComConditions: null,
+    commonlyTreatedCondition: null,
+    vetCenterServiceDescription: 'PTSD care description...',
+    description: 'PTSD care detailed description...',
+    body: '<p>PTSD care body content...</p>',
+  },
+  {
+    name: 'Couples and family counseling',
+    vetCenterTypeOfCare: 'counseling',
+    vetCenterFriendlyName: null,
+    alsoKnownAs: null,
+    vetCenterComConditions: null,
+    commonlyTreatedCondition: null,
```

Comment:
it would be nice to see an example with some of these values filled out. some of the live ones have a bit more content that would be good to see examples of props -> display, e.g.

https://www.va.gov/butler-health-care/locations/abie-abraham-va-clinic/#health-care-offered-here or https://www.va.gov/portland-health-care/locations/portland-va-medical-center/#health-care-offered-here

Reply:
Updated with additional examples, but to be clear, those examples are health care services (src/site/layouts/health_care_local_facility.drupal.liquid), which is a different more extensive component than the vet-center services layout.

Reply:
Ah you are absolutely right, linking to the wrong thing. Sorry about that. Thank you for updating with additional examples.